pdg/src/main.rs

```diff
@@ -1,6 +1,8 @@
 #![feature(min_specialization)]
 #![feature(rustc_private)]
 #![feature(map_try_insert)]
+#![feature(type_ascription)]
```

Comment:
Where is this used?  If we don't need it, which we probably don't, we should avoid it.

Reply:
Sounds good, I think it was just used in a previous version, and I ended up not using it. Will remove.